Sources/Datadog/URLSessionAutoInstrumentation/Interception/URLSessionInterceptor.swift

```diff
@@ -94,6 +94,11 @@ internal class URLSessionInterceptor: URLSessionInterceptorType {
             let interception = TaskInterception(request: request)
             self.interceptionByTask[task] = interception
 
+            if let tracer = Global.sharedTracer as? Tracer,
+               let spanContext = self.extractSpanContext(from: request, using: tracer) {
```

Comment:
in `HTTPHeadersReader.extract()` we have 
```swift
return DDSpanContext(..., parentSpanID: nil, ...)
```
shouldn't we retrieve `activeSpan` here? (this might be out of this PR's scope though)

Reply:
It's a question on the requirement. Do we want auto-instrumented spans to be linked with / continue user-created spans in the app? It was never the goal, so it's not implemented.